### PR TITLE
Add nrrso to SC

### PR DESCRIPTION
**Pull request by aliok**
# Changes

- Add nrrso to SC
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Comment by aliok** [comment]
/hold

@nrrso can you verify the email address please? Is that the one that you would like to use?

File: groups/committee-steering/groups.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ groups:
     owners:
       - aliok@redhat.com
       - evan.k.anderson@gmail.com
-      - puerco@chainguard.dev
       - naina.sng@gmail.com
+      - norris.samosarenkhoe@sva.de
```
Comment (aliok) [inline groups/committee-steering/groups.yaml:13]: Good?

File: groups/groups.yaml [hunk 2]
```diff
@@ -44,8 +44,8 @@ groups:
       - aliok@redhat.com
       - evan.k.anderson@gmail.com
       - lkinglan@redhat.com
-      - puerco@chainguard.dev
       - naina.sng@gmail.com
+      - norris.samosarenkhoe@sva.de
```
Comment (aliok) [inline groups/groups.yaml:48]: Good?

File: groups/committee-steering/groups.yaml (hunk 1 above)
Comment (nrrso) [reply groups/committee-steering/groups.yaml:13]: Please use `developer@nrrso.com` instead :)

File: groups/groups.yaml (hunk 2 above)
Comment (nrrso) [reply groups/groups.yaml:48]: Please use developer@nrrso.com instead :)

**Comment by aliok** [comment]
/unhold

@knative/steering-committee this PR ready to go!

**Comment by dprotaso** [comment]
Hey @nrrso you'll have a bunch of org invites - can you approve those soon. This will unblock owner updates across our repos - eg. https://github.com/knative/func/pull/2229#issuecomment-1996227316

**Comment by dprotaso** [comment]
There will be two invites - one from the `knative` org and the other from `knative-extensions`

**Pull request merged**